pkg/isoeditor/nmstate_handler.go

```diff
@@ -30,52 +30,67 @@ func NewNmstateHandler(workDir string, executer Executer) NmstateHandler {
 	}
 }
 
-func (n *nmstateHandler) CreateNmstateRamDisk(rootfsPath, ramDiskPath string) error {
+func (n *nmstateHandler) CreateNmstateRamDisk(rootfsPath, ramDiskPath, nmstatectlPath string) error {
+	compressedCpio, err := n.buildNmstateCpioArchive(rootfsPath)
+	if err != nil {
+		return err
+	}
+
+	// write for RAM disk
+	err = os.WriteFile(ramDiskPath, compressedCpio, 0755) //nolint:gosec
+	if err != nil {
+		return err
+	}
+
+	// write for caching
```

Comment:
Can you also add to the commit and PR description the reasoning behind why we're doing this with the minimal ISO creation even though caching this has nothing to do with the minimal ISO?